package/plugin-gargoyle-stamgr/files/www/js/sta_manager.js

```diff
@@ -0,0 +1,419 @@
+		{
+			if(req.readyState == 4)
+			{
+				setControlsEnabled(true);
+				window.location =  window.location;
+			}
+		}
+		runAjax("POST", "utility/run_commands.sh", param, stateChangeFunction);
+	}
+}
+
+function resetData()
+{
+	document.getElementById("enablestamgr").checked = uciOriginal.get("gargoyle_stamgr", "global", "enabled") == "1" ? true : false;
+	document.getElementById("maxretry").value = uciOriginal.get("gargoyle_stamgr", "global", "max_retry");
+	document.getElementById("maxwait").value = uciOriginal.get("gargoyle_stamgr", "global", "max_wait");
+	document.getElementById("disconnectqualthresh").value = uciOriginal.get("gargoyle_stamgr", "global", "disconnect_quality_threshold");
+	document.getElementById("connectqualthresh").value = uciOriginal.get("gargoyle_stamgr", "global", "connect_quality_threshold");
+	blacklisttimer = uciOriginal.get("gargoyle_stamgr", "global", "blacklist_timer");
+	document.getElementById("blacklisttimer").value = blacklisttimer;
+
+	//Init STA Table
+	stationTableColumns = [stamgrStr.cfgid, stamgrStr.Band, 'SSID', 'BSSID', stamgrStr.Encryption/*, stamgrStr.Password*/,''];
+	stationTableData = new Array();
+	stationSections = uciOriginal.getAllSectionsOfType('gargoyle_stamgr','stacfg');
+	for(staidx = 0; staidx < stationSections.length; staidx++)
+	{
+		cfgid = stationSections[staidx];
+		
+		band = uciOriginal.get('gargoyle_stamgr',cfgid,'radio');
+		band = radioList[band] == "" ? band : radioList[band];
+		ssid = uciOriginal.get('gargoyle_stamgr',cfgid,'ssid');
+		bssid = uciOriginal.get('gargoyle_stamgr',cfgid,'bssid');
+		bssid = bssid == "" ? "-" : bssid;
+		enc = uciOriginal.get('gargoyle_stamgr',cfgid,'encryption');
+		enc = translateBackendFrontend(enc,encryptionList);
+		pass = uciOriginal.get('gargoyle_stamgr',cfgid,'key');
+		
+		if(band && ssid && enc && (uciOriginal.get('gargoyle_stamgr',cfgid,'encryption') == "none" || pass))
+		{
+			stationTableData.push([cfgid,band,ssid,bssid,enc,pass,createEditButton()]);
+		}
+	}
+
+	stationTable=createTable(stationTableColumns, stationTableData, "wireless_station_table", true, true);
+	stationTableContainer = document.getElementById('wireless_station_container');
+	if(stationTableContainer.firstChild != null)
+	{
+		stationTableContainer.removeChild(stationTableContainer.firstChild);
+	}
+	stationTableContainer.appendChild(stationTable);
+	
+	//Init Runtime Status
+	runtimeObj = JSON.parse(runtime_status == '' ? '{}' : runtime_status);
+	//Fix datetime issues eventually
+	document.getElementById("lastupdate").innerHTML = runtimeObj.lastUpdate === undefined ? "-" : new Date(runtimeObj.lastUpdate*1000).toLocaleString();
+	activecfgid = runtimeObj.active_cfg_id === undefined ? "-" : runtimeObj.active_cfg_id;
+	document.getElementById("currentid").innerHTML = activecfgid;
+	connected = runtimeObj.current_wireless_cfg.connected;
+	document.getElementById("connected").innerHTML = connected == "true" ? "Yes" : "No";
```

Comment:
Not translated (Yes/No).

Reply:
Added Yes/No to strings.js. I don't know how we have avoided them before :)